components/button.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react'
+
+interface Props {
+  text: string
+  type: string
+  size: string
+}
+
+export const Button: FC<Props> = ({ text, type, size }) => {
+  let styleType: string = type + size
```

Comment:
better to use template string

Reply:
If we use a template string instead, the deployment fails

Reply:
mmm that's weird